src/app/pages/send/send-crypto-asset-form/send-crypto-asset-form.routes.tsx

```diff
@@ -32,7 +32,7 @@ const recipientAccountsDrawerRoute = (
 
 const editNonceDrawerRoute = <Route path={RouteUrls.EditNonce} element={<EditNonceDrawer />} />;
 const broadcastErrorDrawerRoute = (
-  <Route path={RouteUrls.TransactionBroadcastError} element={<BroadcastErrorDrawer />} />
+  <Route path={'confirm/broadcast-error'} element={<BroadcastErrorDrawer />} />
```

Comment:
I haven't worked with the send form in awhile and know you have made a lot of changes, but are you sure `{broadcastErrorDrawerRoute}` just doesn't need to be moved and nested in the confirm route here?

Reply:
I tried to do so, but the flow still ends up with error. seems there is problem with sip10/stx routes conflicts bc I added contractId param in sip10 routes